engine/include/cubos/engine/tools/entity_selector/plugin.hpp

```diff
@@ -4,7 +4,7 @@
 
 #include <cubos/engine/cubos.hpp>
 
-namespace cubos::engine::plugins::tools
```

Comment:
Should we keep the `tools` namespace? What do you think=

Reply:
If we remove it that should be done in #338 imo